src/TestUtils/SolutionPathTransformer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpSchool\PhpWorkshop\TestUtils;
+
+class SolutionPathTransformer
+{
+    public static function tempPathToSolutionTempPath(string $tmpFilePath): string
+    {
+        $tmpDir = realpath(sys_get_temp_dir());
+
+        if (!$tmpDir) {
+            throw new \RuntimeException();
```

Comment:
Let's add a message to the exception - also we do have our own `RuntimeException` you could use